src/index.ts

```diff
@@ -326,12 +332,36 @@ const animateDiv = (div: HTMLDivElement, layers: LayerDefinition[], options: Pow
         div.removeChild(div.firstChild);
     }
     // For each animation layer, clone template layer, tweak styles, and append to element
-    for (const layer of layers) {
+    for (let i = 0; i < layers.length; ++ i) {
         const layerDiv = templateLayer.cloneNode(false) as HTMLDivElement;
         layerDiv.style.backgroundImage = `url(${imageUrl})`;
-        layerDiv.animate(layer.steps, layer.timing);
         div.appendChild(layerDiv);
     }
+    // Functions to control animation
+    const startAnimation = () => {
+        layers.forEach((layer, i) => div.children[i].animate(layer.steps, layer.timing));
+    };
+    const stopAnimation = () => {
+        layers.forEach((_, i) => div.children[i].getAnimations().map(animation => animation.cancel()));
+    };
+    switch (options.playMode) {
+        case 'always':
+            startAnimation();
+            div.onmouseenter = null;
```

Comment:
Using onmousenter rather than addEventListener to auto-remove event listeners on each re-glitch. The div is always created inside PowerGlitch so no risk of overriding parent app behavior